Scripts/Core/EditorVR.cs

```diff
@@ -48,8 +46,13 @@ sealed partial class EditorVR : MonoBehaviour
 		DirectSelection m_DirectSelection;
 		Menus m_Menus;
 		UI m_UI;
+		Interfaces m_Interfaces;
+		MiniWorlds m_MiniWorlds;
```

Comment:
Perhaps I wasn't clear enough in the comments. I only needed you to cache the nested modules locally in the nested module that is making use of these lookups during a frequently called method. It isn't necessary outside of that. The ones here only exist because of their use in Update() or because they have to be spawned prior (e.g. m_Interfaces). Please revert any of the new ones and move them locally to a nested module. It might be easier to simply roll back changes to the specific files you changed related to this change, but I'm fine with you leaving whatever modules were already accessible. I implemented GetNestedModule<> primarily to avoid multiple state variables in EVR, but then someone (possibly dylanu) requested that I cache the ones used in Update. Not even sure if that is actually necessary for performance reasons.